### ISSUE TITLE
Estimate to implement new architecture from 3di management

### ISSUE THREAD

**Issue by Derryrover** [nens/lizard-management-client#909]
From Joeri on slack:

https://docs.google.com/document/d/1ceQqPlNZF5cSttF3xqM31AuA0RFFg7jHFvXjhYUkrqM/edit?ts=60fab65f#
Lex heeft verder e.e.a. op papier gezet rondom jullie discussies over de Client Architecture
Zou jij voor mij een inschatting willen maken van wat het zou kosten om die nieuwe opzet te implementeren in Lizard?

Reply Tom:
I think it is best to wait the estimate until Remco is a bit further with this approach.
The estimate is already not so precise. Adding an early estimate will make it even more imprecise.

**Comment by hoanphungt** [comment]
@lexvand @Derryrover @remcogerlich 
I think we can already start with the new Table and Tile from the nens-react-component-library. I can take over this project from Tom and start by making an EPIC for this. 

@lexvand should I include this ticket in the current sprint?

**Comment by Derryrover** [comment]
@lexvand 
@hoanphungt 
@remcogerlich 
@JoepGrispen 

Hi All,

To answer the question:
"Do I think we should now start with refactor the existing management screens to use the new component library?"

My advice is to now look for new projects/applications/features to build instead of refactoring existing ones.
For these new projects we can use Chackra and the component library.

We can for example think about:
- geoblocks final points
- new lizard-client
- klimaat-monitor
- new flooding-client?
- 3di projects?

The reason I think so is too nuanced to explain here completely, but if people wonder we can plan a meeting for me to explain (or for others to convince me I am wrong).
In short I think that refactoring now will not lead to "added value" and we might not even reach our goal of making the codebase more consistent.

kr Tom

**Comment by JoepGrispen** [comment]
I totally agree with Mr Rover

**Comment by hoanphungt** [comment]
@Derryrover 
Yes, it's also a good approach to only use the component library for new new projects/applications. I just think that the lizard-management-client is the best place to start with the component library since the library is made first specifically for the 3Di management interface. So it's logical to start from the the lizard-management-client and that the components in the library can be used there without many changes. Otherwise, it might become the case that the library is hardly used by any other apps apart from the 3Di management and slowly becomes out of date or out of sync with the rest.

Just my thought but yes, I agree that we can start using it for a new project. Maybe for a new management screen in the lizard-management-client (e.g. asset management).
